ui/components/app/modals/contract-details-modal/contract-details-modal.js

```diff
@@ -19,9 +22,26 @@ import {
   SIZES,
   BORDER_STYLE,
 } from '../../../../helpers/constants/design-system';
+import { useCopyToClipboard } from '../../../../hooks/useCopyToClipboard';
+import UrlIcon from '../../../ui/url-icon/url-icon';
+import { getAddressBookEntry } from '../../../../selectors';
 
-export default function ContractDetailsModal({ onClose, address, tokenName }) {
+export default function ContractDetailsModal({
+  onClose,
+  tokenName,
+  tokenAddress,
+  toAddress,
+  chainId,
+  rpcPrefs,
+  origin,
+  siteImage,
+}) {
```

Comment:
Can we add these props to the args in storybook so it doesn't break? Seems to be working nicely on develop [See github pages storybook](https://metamask.github.io/metamask-storybook/index.html?path=/story/ui-components-app-modals-contract-details-modal-contract-details-modal-stories-js--default-story)

<img width="1440" alt="Screen Shot 2022-09-28 at 8 54 25 AM" src="https://user-images.githubusercontent.com/8112138/192832713-cd35487f-d6bd-45a3-92c1-1ac2efb9d989.png">

Reply:
I really need to add storybook checks to the CI 😅 I have a ticket will try get to it this week 🙏